_faq/10-cdi-weld-versions.asciidoc

```diff
@@ -30,19 +30,19 @@ An overview is provided in the following table:
 |1.2
 |7+
 |7
-|Not actively developed anymore.
+|Not actively developed anymore. Support available with link:http://www.jboss.org/products/eap/overview/[JBoss EAP 7.0]
 
 |2.4{set:cellbgcolor:#eee}
 |1.2
 |7+
 |7
-|The stable version of Weld.
+|In maintenance mode. Support available with link:http://www.jboss.org/products/eap/overview/[JBoss EAP 7.1]
```

Comment:
All of these links now redirect to RH developers.
We might as well use `https://developers.redhat.com/products/eap/overview/`.

Reply:
Good point.